.github/workflows/facilitate-incident-review.yml

```diff
@@ -0,0 +1,36 @@
+name: Facilitate Incident Review
+
+on:
+  schedule:
+    - cron: '0 14 * * WED'
+
+jobs:
+  facilitate-incident-review:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    steps:
+      - name: Setup Node.JS
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18'
+
+      - name: Install Dependencies
+        run: npm install --global @artsy/cli
+
+      - name: Run CLI
+        id: cli
+        run: echo "PAYLOAD=$(artsy scheduled:facilitate-incident-review --facilitatorEmail=${{ secrets.EMAIL_OZ }} --useDatesVar)" >> $GITHUB_OUTPUT
```

Comment:
For the first iteration of this, I am assigning myself via `--facilitatorEmail` flag.